packages/mobile/android/app/build.gradle

```diff
@@ -258,7 +258,6 @@ android {
 dependencies {
     androidTestImplementation('com.wix:detox:+')
 
-    implementation 'com.google.android.material:material:1.2.0'
```

Comment:
This was needed to use `.getAttributes()` with detox: https://github.com/valora-inc/wallet/pull/877.

Reply:
oh looks like it's still there but lower in the same block as the new dependencies. Also it got a version bump

Reply:
oh looks like it's still there but lower along with the new dependencies. Also it got a version bump.